about Fern and Bee.html

```diff
@@ -11,36 +11,21 @@
 
   <body>
    
-  <!-- trying to create a horizontal top nav bar to match the one on fernandbee.com -->
-
-  <h1 class="main-title"><a href="./index.html"> FERN AND BEE </a></h1>
-    <ul>
-      <li><a href="./portfolio.html">portfolio</a></li>
-      <li><a href="./blog.html">blog</a></li>
-      <li><a href="./about Fern and Bee.html">about Fern and Bee</a></li>
-      <li><a href="./purchase.html">purchase</a></li>
-      <li><a href="./contact.html">contact</a></li>
+  <!-- navbar -->
+  
+    <ul class="navbar" role="navigation">
+      <h1 class="main-title"><a href="./index.html"> FERN AND BEE </a></h1>
```

Comment:
it is typically not a good idea to nest anything BUT list items in an `ul` or `ol`.

I would make it more like...

```
<div class="navbar">
  <h1 />
  <ul />
</div>
```

and use `.navbar` to do wrapper styling. `.navbar h1` and `.navbar ul` to access the specific h1, ul styling.